lib/nodes/Base.ts

```diff
@@ -152,6 +163,17 @@ export class Base<S extends BlueshellState, E> implements BaseNode<S, E> {
 		this._parent = path;
 	}
 
+	public get parent() {
+		return this._parent;
+	}
+
+	public get id(): string {
+		if (!this._id) {
+			this._id = `n${v4().replace(/\-/g, '')}`;
```

Comment:
what'st he `n` for?

Reply:
@mbittarelli6river it looks like this was in there from the beginning of when we added it in dec 2020 - is that to make it start with a string instead of a number?

Reply:
it was holdover from https://github.com/6RiverSystems/blueshell/pull/147/files#diff-f48473d5b5c7f55d617959eaa22e2bdeac04ddd3fd4d17f7d2bc711a94e6e4b9L60 :)

Reply:
oh it's always been like that? nevermind then